docs/topics/Reference.adoc

```diff
@@ -49,6 +49,8 @@ This section contains full details on advanced features available in KeePassXC.
 |{DB_DIR}	     |Absolute directory path of database file
 |===
 
+NOTE: You can insert literal placeholder strings by escaping the beginning and ending curly braces. For example, to insert the string `{USERNAME}`, you would type `++\{USERNAME\}++`.

```

Comment:
What's the ++ for?

Reply:
Sure about that? That'd for sure be a GitHub markdown parser quirk. 

Test: `\{USERNAME\}`

Reply:
Double \ prints a double \ . Quirky